store/kv/fdb.go

```diff
@@ -584,14 +588,26 @@ func (t *ftx) Commit(ctx context.Context) error {
 	return t.err
 }
 
-func (t *ftx) Rollback(ctx context.Context) error {
+func (t *ftx) Rollback(_ context.Context) error {
 	t.tx.Cancel()
 
 	log.Debug().Msg("tx Rollback")
 
 	return nil
 }
 
+func (t *ftx) End(_ context.Context) error {
+	if t.err != nil {
+		return t.err
+	}
+
+	log.Err(t.err).Msg("tx End")
```

Comment:
Should we log this as error?

Reply:
Actually it should be already logged at the time we get in previous calls.

Reply:
this is some cleanup left, removing.